src/Explorer/Explorer.tsx

```diff
@@ -302,17 +298,11 @@ export default class Explorer {
         this.sendNPSMessage();
       }
     } else {
-      // An existing account is older than 3 days but less than 90 days old. For existing account show to 100% of users in Data Explorer.
+      // Display survey when an existing account is older than 3 days but less than 90 days old. Also, show for accounts greater than 90 days old. For existing account show to 100% of users in Data Explorer.
```

Comment:
Should we remove this part of the comment ? Is it just begin displayed for accounts older than 3 days ?

Reply:
Yes, that makes sense.